cirq-superstaq/cirq_superstaq/service_test.py

```diff
@@ -766,6 +766,43 @@ def test_aces(
     assert service.process_aces("id1") == [1] * 51
 
 
+@mock.patch("requests.post")
+def test_aces_with_weights(
+    mock_post: mock.MagicMock,
+) -> None:
+    service = css.Service(api_key="key", remote_host="http://example.com")
+    with pytest.raises(TypeError, match="Expected weights to be of type Iterable"):
+        service.submit_aces(
+            target="ss_unconstrained_simulator",
+            qubits=[0, 1],
+            shots=100,
+            num_circuits=10,
+            mirror_depth=5,
+            extra_depth=5,
+            noise=cirq.NoiseModel.from_noise_model_like(cirq.depolarize(0.1)),
+            weights=1,  # type: ignore
```

Comment:
Why are we type ignoring here?

Reply:
Those tests check if the methods are handling unexpected types correctly by inputting incorrectly typed weight values. Ingoring the types here so mypy doesn't throw errors while running checks because the incorrect type is intended.